packages/app/src/lib/components/foil/subscribe.tsx

```diff
@@ -241,7 +241,14 @@ const Subscribe: FC<SubscribeProps> = ({
     if (positionId && quoteModifyPositionResult?.error) {
       setQuoteError(quoteModifyPositionResult.error.message);
     } else if (quoteCreatePositionResult.error && !positionId) {
-      const errorMessage = quoteCreatePositionResult.error.message;
+      let errorMessage = quoteCreatePositionResult.error.message;
```

Comment:
I feel like we should have this (in a toast?) in an else on the next if statement?